### ISSUE TITLE
Torch does not have CUDA

### ISSUE THREAD

**Issue by mykeehu** [rsxdalv/tts-generation-webui#356]
I just updated the program on Windows, and I noticed that it has dumbed down torch to no cuda, and `python check_cuda.py` says no cuda, while I have an RTX 3090 card in the machine. I had CUDA before, so I don't understand.

**Comment by rsxdalv** [comment]
Which installer are you using?
Also, if you go into the settings>GPU info tab, what does it show?

**Comment by mykeehu** [comment]
I used update_windows.bat only. And I see this:
![image](https://github.com/user-attachments/assets/11550aa3-34e1-45ab-bf11-f34413deab34)

I will reinstall everything in next days.

**Comment by rsxdalv** [comment]
Yeah, sorry that the old installer broke. I am trying to still maintain some support for it but there are cases where it becomes impossible. The issue I had was - RVC broke (for unknown reason, could not determine it for hours) so I upgraded to a new RVC. The new RVC required a newer torch version, so I upgraded the whole project from 2.0.0 to 2.3.1. However, this was not possible with the old installer, since I would have to make one-click-installers v7 for that change alone. So even while maintaining the old installer script in the repo, it ultimately broke.
